tests/factories.py

```diff
@@ -535,6 +535,7 @@ def _create(cls, target_class, _id, name):
             _id=_id,
```

Comment:
Is it necessary to override `_create` and `_build`? It'd be better if you could just define the default field names, like you normally would with factories:

```python

class InstitutionFactory(ModularOdmFactory):
    FACTORY_FOR = Institution
    _id = Sequence(lambda n: "S{}".format(n))
    name = Sequence(lambda n: "School{}".format(n))
    logo_name = 'logo.img'
    auth_url='http://thisIsUrl.biz'
```